examples/lighting-app/silabs/efr32/args.gni

```diff
@@ -17,6 +17,7 @@ import("${chip_root}/config/standalone/args.gni")
 import("${chip_root}/src/platform/silabs/efr32/args.gni")
 
 efr32_sdk_target = get_label_info(":sdk", "label_no_toolchain")
+efr32_project_dir = "${chip_root}/examples/lighting-app/silabs/efr32"
```

Comment:
what is the difference between this `efr32_project_dir` and the one set at line 34 of `examples/lighting-app/silabs/efr32/BUILD.gn` not they match so it didn't flag an error but we just just keep 1

Reply:
removed